src/actions/constants.js

```diff
@@ -1,2 +1,3 @@
 export const CREATE_BOOK = 'CREATE_BOOK';
```

Comment:
## Optional Change
- [ ] It would be great if the title and the category field were validated for empty `field-checks` to discourage the creation of `empty` tittle and `empty` category as shown below. 👇 Kindly enforce this rule in these aforementioned fields. Thank you!

![empty-book-creation](https://user-images.githubusercontent.com/22345961/107087015-df95d680-680b-11eb-92f6-f31496ed4024.gif)